CONTRIBUTING.md

```diff
@@ -4,22 +4,31 @@
 
 ## Table of contents
 
-- [Overview](#overview)
-- [Setting up the development environment](#setting-up-the-development-environment)
-	- [Install](#install)
-	- [Run](#run)
-	- [Test](#test)
-- [Adding new resources](#adding-new-resources)
-	- [Glossary](#glossary)
-	- [Quickstart](#quickstart)
-	- [Cost component names and units](#cost-component-names-and-units)
-	- [Finding prices](#finding-prices)
-	- [Adding usage-based cost components](#adding-usage-based-cost-components)
-	- [General guidelines](#general-guidelines)
+- [Contributing to Infracost](#contributing-to-infracost)
+	- [Table of contents](#table-of-contents)
```

Comment:
Not sure we need a link to this since we are already in the Table of contents here